### PR TITLE
Fix section 6.1.5

### PR DESCRIPTION
**Pull request by kroeckx**
Github's markdown requires 3 dashes, kramdown works with any

This also adds the header to each tables, as they are in the pdf file.

### CODE REVIEW

**Comment by sleevi** [comment]
I'm sorry, this change both doesn't work and contains material changes that would need to be balloted, such as updating the text and references.

**Comment by pzb** [comment]
It looks like the build is broken for an unrelated reason.  Additionally the changes Kurt made match the current BRs.  Both https://cabforum.org/wp-content/uploads/BRv1.2.5.pdf and https://cabforum.org/wp-content/uploads/CA-Browser-Forum-BR-1.3.4.pdf have the reference to FIPS 186-4.  The markdown version does not match the official version here.

**Comment by sleevi** [comment]
Ah, yeah, I broke the .travis.yml build for PIP/WeasyPrint when merging #17. Separate from that, through, changing the footnote in an issue about fixing tables seems... Not right.

**Comment by pzb** [comment]
I just changed this topic to fixing 6.1.5.  Once the build works again, we can compare the current official 6.1.5 to both the old Appendix A and to the HTML/PDF build from markdown and see if this PR has any net changes.

**Comment by kroeckx** [comment]
After fixing one thing, I saw other things that didn't look right, so I think the combination of those patches should make it match with the official BR docs.

File: docs/BR.md [hunk 1]
```diff
@@ -1018,31 +1018,35 @@ Certificates MUST meet the following requirements for algorithm type and key siz
...
+|Digest algorithm|SHA-1, SHA-256, SHA-384 or SHA-512|SHA-1*, SHA-256, SHA-384 or SHA-512|
+|Minimum RSA modulus size (bits)|1024|2048|
+|ECC curve|NIST P-256, P-384, or P-521|NIST P-256, P-384, or P-521|
+|Minimum DSA modulus and divisor size (bits)\*\*\*|L= 2048, N= 224 or  L= 2048, N= 256|L= 2048 N= 224 or L= 2048 N= 256
 
 (3) Subscriber Certificates
 
+||Validity period *ending* on or before 31 Dec 2013|Validity period *ending* after 31 Dec 2013|
+|---|---|---|
 |Digest algorithm|SHA-1*, SHA-256, SHA-384 or SHA-512|
```
Comment (pzb) [inline docs/BR.md]: This row is still wrong.  The last column is blank but should be `SHA1*,	SHA‐256,	SHA‐384	or	SHA‐
512	`

**Comment by pzb** [comment]
From my review, the revised version matches the current official version of the BRs with the noted exception.  In my opinion, this should be good to merge if the missing cell is fixed.

File: docs/BR.md (hunk 1 above)
Comment (kroeckx) [inline docs/BR.md]: That's fixed in the next patch?

Maybe I should squash them?

Comment (kroeckx) [inline docs/BR.md]: Oh, I see now ...

**Comment by kroeckx** [comment]
I think everything should be fixed now.

**Comment by pzb** [comment]
Looks good to me now.  Can someone else double check and merge?

**Pull request merged**